client/me/concierge/shared/upsell.js

```diff
@@ -25,13 +25,7 @@ class Upsell extends Component {
 					<Site siteId={ site.ID } />
 				</CompactCard>
 				<CompactCard>
-					<p>
-						{ translate( 'You do not have any available Quick Start sessions.' ) }{ ' ' }
-						{ translate( 'Click the button to purchase a new session.' ) }
-					</p>
-					<Button href={ `/checkout/offer-quickstart-session/${ site.slug }` } primary>
-						{ translate( 'Learn More' ) }
-					</Button>
+					<span>{ translate( 'You do not have any available Quick Start sessions.' ) }</span>
```

Comment:
I'm curious, what's the reason for switching from `<p>` to `<span>`?

Reply:
The `<p>` was adding margin (or padding) below the text, which made the vertical spacing of the card really odd-looking. That wasn't a problem before when there was a button, but it looked really weird without the button.